.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: generic
```

Comment:
I've not tried using travis-ci w/ an open source project before, but I absolutely love that you've added this :)

Is there anything that will needs to be configured at travis-ci.com to make this work?

Reply:
Yes you need to generate an API-Token on Docker Hub and add that to the Travis Project via the HUB_TOKEN Environment Variable and you also need to set HUB_USER to you accoutname.

Reply:
Ok, I kinda love this idea :)

However, I need a bit to get this all wired up. It also makes for multiple large and only semi related changes in the PR. How would you feel about pulling the `travis.yml` file out into a separate PR? Everything else LGTM and I'd love to be able to merge it asap.

Reply:
Oh, sorry. One other comment on your last comment above. I currently have it configured so DockerHub watches GitHub for changes and automatically does this build there. I think that's the same thing you were going for via Travis. Any advantages to moving that build out of DockerHub and into Travis?